From: David Chirgwin <dchirgwin@bluechiptechnology.co.uk>
Date: Wed, 24 Sep 2014 10:36:47 +0100
Subject: [PATCH] Remove call to xf86XInputSetScreen()

============================================================
--- a/src/tslib.c
+++ b/src/tslib.c
@@ -194,10 +194,6 @@ static void ReadInput (InputInfoPtr local)
 			x = samp.x;
 			y = samp.y;
 
-			xf86XInputSetScreen(local, priv->screen_num,
-					samp.x,
-					samp.y);
-
 			xf86PostMotionEvent (local->dev, TRUE, 0, 2,
 					x, y);
 

