From cf5ebce899f1b4963f4e15bcfe5d9c6986890448 Mon Sep 17 00:00:00 2001
From: David Chirgwin <dchirgwin@bluechiptechnology.co.uk>
Date: Mon, 13 Oct 2014 16:46:59 +0100
Subject: [PATCH] Remove desktop shortcut for matchboard keyboard

---
 Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 427206e..3e902c7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,9 +12,6 @@ if WANT_APPLET
 SUBDIRS += applet
 endif 
 
-desktopdir = $(datadir)/applications/inputmethods
-dist_desktop_DATA = matchbox-keyboard.desktop
-
 pixmapsdir = $(datadir)/pixmaps/
 dist_pixmaps_DATA = matchbox-keyboard.png
 
-- 
1.9.1

